src/sentry/integrations/github/integration.py

```diff
@@ -106,8 +106,11 @@ class GitHubIntegration(IntegrationInstallation, GitHubIssueBasic, RepositoryMix
     def get_client(self) -> GitHubClientMixin:
         return GitHubAppsClient(integration=self.model)
 
-    def get_trees_for_org(self) -> JSONData:
-        return self.get_client().get_trees_for_org(self.model.name)
+    def get_trees_for_org(self, cache_seconds: int = 3600 * 24) -> JSONData:
+        return self.get_client().get_trees_for_org(
+            gh_org=self.model.metadata["domain_name"].split("github.com/")[1],
```

Comment:
might be cleaner to define this in a variable and then use that as the param. No strong feelings either way